images_and_binaries.sh

```diff
@@ -13,10 +13,10 @@ declare -A RHCOS_BOOT_IMAGES
 # Map of boot type to raw metal image
 declare -A RHCOS_METAL_IMAGES
 
-# 2/6/2020: 4.4 must be acquired from internal Red Hat CI registry, because it is not yet available 
+# 2/6/2020: 4.4/4.5 must be acquired from internal Red Hat CI registry, because it is not yet available 
 # from the official mirror
-if [ "$OPENSHIFT_RHCOS_MAJOR_REL" == "4.4" ]; then
-    BUILDS_JSON="$(curl -sS https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-$OPENSHIFT_RHCOS_MAJOR_REL/builds.json)"
+if [ "$OPENSHIFT_RHCOS_MAJOR_REL" == "4.4" ] || [ "$OPENSHIFT_RHCOS_MAJOR_REL" == "4.5" ]; then
+    BUILDS_JSON="$(curl -sS https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/builds.json)"
```

Comment:
We would need to differentiate here between 4.4 and 4.5 in this URL.

Reply:
+1, seems 4.5 rhcos url is available now, I will change it.